react/components/admin/institutional/README.md

```diff
@@ -0,0 +1,28 @@
+# VTEX Content Pages
```

Comment:
All pages with content are technically "Content Pages", so I think this may be a bit misleading; perhaps it'd be better to stick to the term "institutional".

@thiagomurakami, what do you think?

Reply:
Institutional was a bad choice actually. `Institutional` refers to institutions, like universities. `Content Page` is the name they give to this kind of page, but I'm open to suggestions.

Reply:
Maybe `Custom Content`? I agree with @alcar that technically all pages are "content pages"..

Btw, I think you should create a `docs` folder on the project root and add this file there. Also, add a link on the project's `README` if possible :)

Reply:
Hi @augustocb , we're calling these pages ad hoc, "standard pages". I think you can use the same label as us.